Release Fireworks model template (#233)
vellum-ai/vellum-client-generator: fern/docs/content/change-log/2024-04.mdx

title: Changelog | April, 2024
---

## Fireworks Finetuned Models

_April 24th, 2024_

> We now support models that you've finetuned on [Fireworks AI](https://fireworks.ai/) in Vellum! You can add your finetuned Fireworks model to your workspace by navigating to the [Models](https://app.vellum.ai) page and clicking on the featured Model template at the top.

Reviewer: ```suggestion
Vellum now supports models that you've fine-tuned on [Fireworks AI](https://fireworks.ai/). You can add your fine-tuned Fireworks model by navigating to the [Models page](https://app.vellum.ai/models) and clicking on the featured model template at the top.
```

![Fireworks Model Template](https://storage.googleapis.com/vellum-public/help-docs/fireworks_model_template.png)

> Note that we currently only support the Mistral models as the base model. If there are other base models that you would like to see supported, please reach out to us!

Reviewer: ```suggestion
Note that only the Mistral family of models are supported currently. If there are other base models that you would like to see supported, please reach out to us!
```
